lib/widget/youtube/youtubestate.dart

```diff
@@ -0,0 +1,7 @@
+import 'package:ensemble/widget/youtube/native/youtube_platform_native.dart'
```

Comment:
this doesn't seem right as you are always importing the native libraries. So the native will be imported even for web? is it import or export?

just look at lib/widget/webview/webviewstate.dart to see how it should be done

Reply:
To fix the issue where the youtubeplayer doesn't work sometimes on the ensemble web studio. This issue is actually an issue from the package side. 
https://github.com/sarbagyastha/youtube_player_flutter/issues/819

In the issue, it was said to load the WebYoutubePlayerIframePlatform(). This requires importing libraries that work on the web only. 

The native library doesn't get imported for the web. It is actually done in the same way as the lottie widget. 
```
export 'native/lottiestate.dart' if (dart.library.html) 'web/lottiestate.dart';
```
I have tested multiple youtube widgets in the same screen. It doesn't load the video in the player in web. That is why I added the videoList for loading the multiple video in a single player. This problem exists in the demo that they provided. https://sarbagyastha.com.np/youtube_player_flutter/#/